draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -226,11 +238,13 @@ This enables clients to avoid sending the user to a dead end, by only redirectin
 
 In addition to the security considerations in OAuth 2.0 Core {{RFC6749}}, and OAuth 2.0 Threat Model and Security Considerations {{RFC6819}}, and {{I-D.draft-ietf-oauth-security-topics}} the additional considerations apply.
 
-## Client ID Metadata Documents for Development Purposes {#documents_for_development}
+## Relationship between `redirect_uris` and `client_id` or `client_uri` {#redirect_uri_relationship}
+
+An authorization server MAY impose no restriction or relationship between the `redirect_uris` and the `client_id` or `client_uri` properties, allowing for a Client ID Metadata Document to be hosted anywhere on the public web, however, this comes with security considerations of potential trust and safety issues, where the client attempts to impersonate a more well-known client or otherwise act in a way which is malicious or puts the end-user at risk.
```

Comment:
I don't think there's anything before this that suggests there would be a restriction, so it seems backwards to say it "MAY impose no restriction". It'd be better to flip this around and say it "MAY impose a restriction .. such as .." and then refer to the development section above.